home_audit.node.js

```diff
@@ -1306,6 +1306,23 @@ let validationRules = {
             );
         }
     },
+
+
+    _is_servicing_duct_system: function(value, sys, duct) {
```

Comment:
Can you add a comment explaining what this does?

Reply:
JSDoc comment added to signature.